upip/bootstrap_upip.sh

```diff
@@ -9,7 +9,7 @@ fi
 
 # Remove any stale old version
 rm -rf micropython-upip-*
-wget -nd -r -l1 https://pypi.python.org/pypi/micropython-upip/ --accept-regex ".*pypi.python.org/packages/source/.*.gz" --reject=html
+wget -nd -rH -l1 -D files.pythonhosted.org https://pypi.org/project/micropython-upip/ --reject=html
```

Comment:
Does it need the pythonhosted.org bit?

Reply:
Yes, we only want to download links with the `files.pythonhosted.org` host because that's where the distributions  live (hence the `--accept-regex` no longer being necessary)